zppy/templates/e3sm_diags.bash

```diff
@@ -40,7 +40,8 @@ tag="{{ tag }}"
 results_dir=${tag}_${Y1}-${Y2}
 
 # Create temporary workdir
-workdir=`mktemp -d tmp.${id}.XXXX`
+hash=`mktemp --dry-run -d XXXX`
+workdir=`mkdir tmp.{{ prefix }}.${id}.${hash}`
```

Comment:
I'm getting errors in `/lcrc/group/e3sm/ac.forsyth2/zppy_min_case_nco_output/test-636-20241209/v3.LR.historical_0051/post/scripts`, so this can't be sufficient. Will need to debug more what's going on.